docs/templates/snakemake/config/config.[METHOD].yaml

```diff
@@ -0,0 +1,13 @@
+# Configuration file for snakemake pipeline
+
+# Dataset samples
+samples: "config/[SAMPLES].csv"
+out_dir: "PATH/TO/OUTDIR"
```

Comment:
Maybe clearer to say `PATH/TO/OUTDIR` where `OUTDIR` should be of the form `IDENTIFIER_METHOD`

Reply:
included this, and also that it is DATASET dependent.